### ISSUE TITLE
New issue occured (AttributeError: 'NoneType' object has no attribute 'get')

### ISSUE THREAD

**Issue by 2easyjunmin** [gochendong/suno-api#22]
session_id = resp.json().get("response").get("last_active_session_id")
AttributeError: 'NoneType' object has no attribute 'get'

Anyone have a solution?

**Comment by purerosefallen** [comment]
Something changed on Suno side?

**Comment by 2easyjunmin** [comment]
yeah, 401 Unauthorized erro rreturn when "https://clerk.suno.com/v1/client?.."

**Comment by gochendong** [comment]
I use paid Acedata now

**Comment by 2easyjunmin** [comment]
Did you fix anything else?
Can you give us a quick explanation of how you did it? @gochendong

**Comment by 2easyjunmin** [comment]
Thank you. You didn't change any other code, did you?
I'd like to get back up and running quickly and would like to be sure.
Can you give me some advice on what you did with acedata.cloud and how you applied it? @gochendong

**Comment by gochendong** [comment]
> Thank you. You didn't change any other code, did you?谢谢你没改别的密码吧？ I'd like to get back up and running quickly and would like to be sure.我想尽快恢复并运行，并希望得到肯定。 Can you give me some advice on what you did with acedata.cloud and how you applied it? 你能给我一些关于你用acedata.cloud做了什么以及你如何应用它的建议吗？@gochendong

Please check the README file. You need to obtain a new token. It is paid, but there are some free quotas available.

**Comment by 2easyjunmin** [comment]
Where can I find this token?
I don't see it anywhere after logging into the site.
https://platform.acedata.cloud/console/platform-tokens
Could this be a token of the following type at this address?


'platform-v1-fb9....'
@gochendong

**Comment by 2easyjunmin** [comment]
Thanks!
But the code is still not under our control,
and we can't 100% trust that the sites we use as third parties will respond well. @gochendong

**Comment by gochendong** [comment]
> Sorry, I don't get it. Where can I acquire the token on the linked website? Is it the platform token?

https://platform.acedata.cloud/services/suno

**Comment by Fumble2** [comment]
It's ok that you want to farm referral, but the service literally doesn't work for suno, I tried to create songs on the site and it just says in a loop of creation(nothing happens). I don't see a reason ruin your code to change it to a service that doesn't work.

https://i.imgur.com/QwcHb9Z.png

**Comment by Legus-Yeung** [comment]
> Thanks! But the code is still not under our control, and we can't 100% trust that the sites we use as third parties will respond well. @gochendong

Yeah, the most recent change is completely reliant on acedata which may or may not work at anytime. Not the best solution.

**Comment by gochendong** [comment]
> > Thanks! But the code is still not under our control, and we can't 100% trust that the sites we use as third parties will respond well. 谢谢你，谢谢但是代码仍然不在我们的控制之下，我们不能100%相信我们作为第三方使用的网站会响应良好。@gochendong
> 
> Yeah, the most recent change is completely reliant on acedata which may or may not work at anytime. Not the best solution.是的，最近的变化是完全依赖于acedata，这可能会或可能不会在任何时候工作。不是最好的解决办法。

The current cookie-based code is completely unusable. Between unusable and unreliable, I choose unreliable. Code that cannot run is the worst code, this is my principle.

If there are better alternatives to replace Acedata, please let me know. Thank you.

**Comment by angejr** [comment]
How do you obtain the token after going to https://surl.id/1uHHTdiT2F ? 
I tried the inviter_id appaering on the url : https://platform.acedata.cloud/?inviter_id=xxxxxxxx
But I get a timeout when trying to generate songs
